index.html

```diff
@@ -284,6 +286,19 @@ <h2>Methods</h2>
                   <li><p>If there is no audio output device, reject
                   <var>p</var> with a new <code>DOMException</code> whose <code>name</code> attribute
                   has the value <code>NotFoundError</code>.</p></li>
+                  <li>
+                    <p>If <var>deviceId</code> is not <code>""</code>, and
+                    matches an id previously exposed by
+                    <code>selectAudioOutput</code> in an earlier browsing
+                    session, the user agent MAY decide to run the following
+                    sub steps:</p>
+                    <ol>
+                      <li>
+                        <p>Let <var>device</var> be the device identified by <var>deviceId</var>.</p>
+                      </li>
+                      <li><p>Resolve <var>p</var> with either <var>deviceInfo</var> and abort all steps.</p></li>
+                    </ol>
+                  </li>
```

Comment:
Should add a "class=fingerprint" Note section here and explain why this MAY is a Good Idea (rather than MUST).